backend/src/main/java/com/happy/friendogly/club/controller/ClubController.java

```diff
@@ -48,7 +50,17 @@ public ApiResponse<List<FindClubByFilterResponse>> findByFilter(
             @Auth Long memberId,
             @Valid @ModelAttribute FindClubByFilterRequest request
     ) {
-        return ApiResponse.ofSuccess(clubQueryService.findFindByFilter(memberId, request));
+        return ApiResponse.ofSuccess(clubQueryService.findByFilter(memberId, request));
+    }
+
+    @GetMapping("/mine")
+    public ApiResponse<List<FindClubMineResponse>> findMine(@Auth Long memberId){
+        return ApiResponse.ofSuccess(clubQueryService.findMine(memberId));
+    }
+
```

Comment:
`my club`이라는 이름이 내가 참여한 클럽인지, 내가 소유한 클럽인지 더 명시적이면 좋을 것 같아요.
아래 participating은 명시적이라서 좋네요.

```suggestion

    @GetMapping("/owning")
    public ApiResponse<List<FindClubMineResponse>> findOwning(@Auth Long memberId){
        return ApiResponse.ofSuccess(clubQueryService.findOwning(memberId));
    }

```